.github/workflows/ci.yml

```diff
@@ -473,3 +480,65 @@ jobs:
         with:
           name: droid-${{ matrix.api-level }}-testapp${{ matrix.unity-version }}-screenshot
           path: samples/artifacts/builds/Android/screen.png
+
+  ios-smoke-test-build:
+    needs: [build]
+    name: Build Smoke Test Ipa - Unity ${{ matrix.unity-version }}
```

Comment:
iOS App Store Package, but after reading the name again, I think it might be better to just keep it as 
```suggestion
    name: Build Smoke Test - Unity ${{ matrix.unity-version }}
```